copc/index.qmd

```diff
@@ -0,0 +1,12 @@
+---
+title: Cloud-Optimized Point Clouds (COPC)
+subtitle: An Introduction to Cloud-Optimized Point Clouds (COPC)
+---
+
+The LAS(er) file format is designed to store 3-dimension point cloud data typically collected from [LiDAR](https://en.wikipedia.org/wiki/Lidar).
+
+A COPC file is a valid LAZ file. An LAZ file is a compressed LAS file. The LAS file format stores 3-dimension (x,y,z) point cloud data, typically collected from LiDAR. COPC files are similar to COGs for GeoTIFFs: Both are valid versions of the original file format but with additional requirements to support cloud-optimized data access. In the case of COGs, there are additional requirements for tiling and overviews. For COPC, data must be organized into a clustered octree with a variable-length record (VLR) describing the octree structure.
```

Comment:
Nit: I'd switch the first two sentences to build from LAS -> LAZ -> COPC. The third sentence is an exact copy of the first sentence, but maybe it bears repeating